docs/source/docs/quick-start/networking.md

```diff
@@ -2,28 +2,49 @@
 
 ## Physical Networking
```

Comment:
We really should just defer to the vivid hosting docs and not rehost the same information that they talk about, or at least link back.

Reply:
It does link back. Also technically they don't show wiring a coprocessor without a switch.